ts/data/project.ts

```diff
@@ -0,0 +1,127 @@
+import { legendAction, TableDefinition } from "../component/table-builder";
+import { makeNameDiv } from "../util/html-utils";
+import { urls } from "../util/urls";
+import { ProjectSummary } from "./case";
+
+export const projectDefinition: TableDefinition<ProjectSummary, void> = {
+  queryUrl: urls.rest.projects,
+  defaultSort: {
+    columnTitle: "Name",
+    descending: true,
+    type: "text",
+  },
+  staticActions: [legendAction],
+  filters: [
+    {
+      title: "Name",
+      key: "NAME",
+      type: "text",
+      autocompleteUrl: urls.rest.autocomplete.projectNames,
+    },
+  ],
+  generateColumns(data) {
+    return [
+      {
+        title: "Name",
+        sortType: "text",
+        addParentContents(projectSummary, fragment) {
+          fragment.appendChild(
+            makeNameDiv(
+              projectSummary.name,
+              urls.miso.project(projectSummary.name),
+              urls.dimsum.project(projectSummary.name)
+            )
+          );
+        },
+      },
+      {
+        title: "Tests Expected",
+        addParentContents(projectSummary, fragment) {
+          fragment.appendChild(
+            document.createTextNode(projectSummary.totalTestCount.toString())
+          );
+        },
+      },
+      {
+        title: "Receipt",
```

Comment:
Please rename the QC gate columns:

* Received
* Extracted
* Libraries Prepared
* Libraries Qualified
* Sequenced
* Informatics Reviewed
* Report Drafted
* Report Finalized

I think these names more clearly indicate that they're counting completed tests